### PR TITLE
SAMBRO: handle multiple info in profile + SAMBRO:tweaks

### PR DESCRIPTION
**Pull request by biplovbhandari**
@flavour Please merge..

### CODE REVIEW

File: controllers/cap.py [hunk 1]
```diff
@@ -388,38 +388,44 @@ def prep(r):
...
+                            query = (ptable.info_id == info["cap_info.id"])
+                        parameters = db(query).select(ptable.name,
+                                                      ptable.value)
 
                     # Area
-                    # @ToDo: handle multiple areas
-                    atable = s3db.cap_area
-                    area = db(atable.alert_id == alert_id).select(atable.name,
-                                                                  limitby=(0, 1)).first()
+                    data = resource.select(["area.name"], raw_data=True)
```
Comment (flavour) [inline controllers/cap.py]: Why do a separate select here? Why not just add this field to the 1st one?

File: controllers/cap.py [hunk 2]
```diff
@@ -662,7 +667,7 @@ def qualifiers_widget(r, **attr):
                 # Check for prepopulate
                 if alert_id:
                     itable.web.default = settings.get_base_public_url()+\
-                                         URL(c="cap", f="alert", args=alert_id)
+                                         URL(c="cap", f="public", args=alert_id)
```
Comment (flavour) [inline controllers/cap.py]: Is this true for all Alerts or do we need to distinguish?

File: modules/s3db/cap.py [hunk 3]
```diff
@@ -1759,7 +1760,7 @@ def cap_info_onaccept(form):
             #    set_.update(parameter = fstring)
 
             web = "%s%s" % (current.deployment_settings.get_base_public_url(),
-                            URL(c="cap", f="alert", args=[alert_id]))
+                            URL(c="cap", f="public", args=[alert_id]))
```
Comment (flavour) [inline modules/s3db/cap.py]: Same question

**Comment by biplovbhandari** [comment]
@flavour Corrected please merge

**Pull request merged**